config/optional/jsonapi_extras.jsonapi_resource_config.paragraph--call_to_action_image.yml

```diff
@@ -0,0 +1,124 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - paragraphs.paragraphs_type.call_to_action_image
+id: paragraph--call_to_action_image
+disabled: false
+path: paragraph/call_to_action_image
+resourceType: paragraph--call_to_action_image
+resourceFields:
+  id:
+    fieldName: id
+    publicName: id
+    enhancer:
+      id: ''
+    disabled: false
+  uuid:
+    fieldName: uuid
+    publicName: uuid
+    enhancer:
+      id: ''
+    disabled: false
+  revision_id:
+    fieldName: revision_id
+    publicName: revision_id
+    enhancer:
+      id: ''
+    disabled: false
+  type:
+    fieldName: type
+    publicName: type
+    enhancer:
+      id: ''
+    disabled: false
+  langcode:
+    fieldName: langcode
+    publicName: langcode
+    enhancer:
+      id: ''
+    disabled: false
+  uid:
+    fieldName: uid
+    publicName: uid
+    enhancer:
+      id: ''
+    disabled: false
+  status:
+    fieldName: status
+    publicName: status
+    enhancer:
+      id: ''
+    disabled: false
+  created:
```

Comment:
Every time a new resource is added a date_time enhancer should be used for any date_time fields See https://github.com/dpc-sdp/content-vic-gov-au/blob/fc6b2f92d4a634bd7bdcb7ee4ab19d3ec0533a53/config/sync/jsonapi_extras.jsonapi_resource_config.node--news.yml#L100

Reply:
Changed applied